packages/url_launcher/url_launcher_linux/linux/url_launcher_plugin.cc

```diff
@@ -45,6 +45,16 @@ static gchar* get_url(FlValue* args, GError** error) {
   return g_strdup(fl_value_get_string(url_value));
 }
 
+// Check if URI has launchable file resource
```

Comment:
Nit: s/Check/Checks/, and add the missing period, to follow Google style.

Reply:
Fixed